### DISCUSSION TITLE
有没有在线的demo可以体验？

### DISCUSSION THREAD

**Discussion by alsotang** [Tencent/cherry-markdown#618] [Q&A] [answered]
这种编辑器按理说是可以做个在线demo，左侧编辑区域，右侧预览的。
看到readme里面有个 `run on cloud studio`，点进去竟然是个在线editor，我还以为是demo。

**Comment by RSS1102** [comment]
你好，可以查看以下demo体验：https://github.com/Tencent/cherry-markdown#demos

**Comment by sunsonliu** [comment, marked as answer]
https://github.com/Tencent/cherry-markdown#demos
readme里有在线demo的入口哈

**Reply by alsotang** [reply]
sorry是我没仔细看。
